doc/data/api/ttn.lorawan.v3/enums.yml

```diff
@@ -252,71 +252,71 @@ DownlinkPathConstraint:
   values:
   - name: DOWNLINK_PATH_CONSTRAINT_NONE
     comment: |2
-       Indicates that the gateway can be selected for downlink without constraints by the Network Server.
+      Indicates that the gateway can be selected for downlink without constraints by the Network Server.
```

Comment:
This is generated; is the generation fixed as well?

Reply:
Yeah this is indeed generated. I'll take a look at the generator.

Reply:
@rvolosatovs you can cherry-pick add87c594aebe3a507b38e39d33b240583faaa49 for new generated data

Reply:
Is there still something to be cherry-picked here?